src/signals/IncidentMap/components/FilterPanel/FilterPanel.tsx

```diff
@@ -56,19 +56,13 @@ export const FilterPanel = ({ filters, setFilters, setMapMessage }: Props) => {
     <>
       <Heading as="h4">Filter op onderwerp</Heading>
```

Comment:
The filter panel component is three levels deep now while the top level doesn't do much, and FilterCategory a lot. Maybe rearrange them as: 

```
<FilterPanel>
filters.map(

if with subCategories --> <FilterWithDropDown />
else --> <Filter />

)
</FilterPanel >
```

Reply:
Done, but now the code is less generic. It will only work for subCategories and mainCategories, if the subCategories ever will have subCategories of their own we're out of luck.